src/pages/index.js

```diff
@@ -1,30 +1,39 @@
 import "./index.css";
-import { config, formValidators, initialCards } from "../utils/constants.js";
+import { config, formValidators } from "../utils/constants.js";
+import Api from "../components/Api.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupWithForm";
 import PopupWithImage from "../components/PopupWithImage";
+import PopupConfirm from "../components/PopupConfirm.js";
 import UserInfo from "../components/UserInfo";
 
 /*══════╕
  │ USER │
  ╘══════*/
 const user = new UserInfo(".profile__title", ".profile__description");
+const profileImage = document.querySelector(".profile__image");
 
 /*══════════════╕
  │ PAGE BUTTONS │
  ╘══════════════*/
-export const editButton = document.querySelector(".profile__edit-button");
-export const addButton = document.querySelector(".profile__add-button");
+export const editProfileButton = document.querySelector(".profile__edit-button");
+export const editImageButton = document.querySelector(".profile__image-update");
+export const addCardButton = document.querySelector(".profile__add-button");
 
-editButton.addEventListener("click", () => {
+editProfileButton.addEventListener("click", () => {
   const { name, occupation } = user.getUserInfo();
+  editProfilePopup.saving(false);
   editProfilePopup.open();
   editProfilePopup.setInputValues({ title: name, description: occupation });
   formValidators["edit-profile-form"].resetValidation();
 });
-addButton.addEventListener("click", () => {
+editImageButton.addEventListener("click", () => {
+  editProfileImagePopup.saving(false);
+  editProfileImagePopup.open();
```

Comment:
Please, call `resetValidation` for tha avatar as well to disable the button on opening